CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+x.x.x
+------
+
+**CHANGES**
+- Disable deeper C states in x86_64 AMI and build-image to guarantee high performance and lower latency
```

Comment:
What about:
`Disable deeper C-States in x86_64 official AMIs and AMIs created through `build-image` command,  to guarantee high performance and low latency.`

Anyway:
- C states --> C-States
- AMI --> AMIs
- build-image --> is a command, let's use backticks
- lower --> low ?
- missing final dot.